components/requests/modals/EditDoctorInformationModal.tsx

```diff
@@ -15,62 +15,50 @@ import {
   FormHelperText,
   Box,
   Divider,
-  useToast,
 } from '@chakra-ui/react'; // Chakra UI
-import SuccessfulEditAlert from '@components/permit-holders/SuccessfulEditAlert'; // Successful edit alert/toast
-import { useState, useEffect, SyntheticEvent, ReactNode } from 'react'; // React
+import { useState, SyntheticEvent, ReactNode } from 'react'; // React
 import { DoctorInformationCardPhysician } from '@tools/components/internal/requests/doctor-information-card'; // Physician type
+import { UpsertPhysicianInput } from '@lib/graphql/types'; // Upsert physician type
 
 type EditDoctorInformationModalProps = {
   children: ReactNode;
   readonly physician: DoctorInformationCardPhysician;
-  readonly onSave: (applicationData: any) => void;
+  readonly onSave: (physicianData: UpsertPhysicianInput) => void;
 };
 
 export default function EditDoctorInformationModal({
   children,
-  physician,
   onSave,
 }: EditDoctorInformationModalProps) {
   const { isOpen, onOpen, onClose } = useDisclosure();
 
   const [name, setName] = useState('');
-  const [mspNumber, setMspNumber] = useState<number | undefined>();
+  const [mspNumber, setMspNumber] = useState(0);
   const [phoneNumber, setPhoneNumber] = useState('');
   const [addressLine1, setAddressLine1] = useState('');
   const [addressLine2, setAddressLine2] = useState<string | undefined>('');
   const [city, setCity] = useState('');
   const [postalCode, setPostalCode] = useState('');
 
-  const successfulEditToast = useToast();
-  useEffect(() => {
-    setName(physician.name);
-    setMspNumber(physician.mspNumber);
-    setPhoneNumber(physician.phone);
-    setAddressLine1(physician.addressLine1);
-    setAddressLine2(physician.addressLine2 || undefined);
-    setCity(physician.city);
-    setPostalCode(physician.postalCode);
-  }, [physician]);
+  let loading = false;
```

Comment:
Same thing with `loading` as commented in `components/permit-holders/modals/EditUserInformationModal.tsx`